src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carRentCosts = 40;
+
+  if (days <= 2) {
+    return carRentCosts * days;
+  } else if (days >= 7) {
+    return days * carRentCosts - 50;
```

Comment:
Move 50 and 20 to consts too
It's better to keep all your numbers in named variables/consts, then it's obvious what this number means